transport/transport.go

```diff
@@ -63,13 +63,11 @@ type recvMsg struct {
 	err error
 }
 
-func (recvMsg) isItem() bool {
-	return true
-}
+func (*recvMsg) Item() {}
 
 // All items in an out of a recvBuffer should be the same type.
 type item interface {
-	isItem() bool
+	Item()
```

Comment:
done

On Sat, Apr 2, 2016 at 2:15 AM, Qi Zhao <notifications@github.com> wrote:

> In transport/transport.go
> <https://github.com/grpc/grpc-go/pull/615#discussion_r58289418>:
>
> >
> >  // All items in an out of a recvBuffer should be the same type.
> >  type item interface {
> > -	isItem() bool
> > +	Item()
>
> unexported
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/615/files/55b5447f28b48ddb0fd04e7ea605fce7d4a425ac#r58289418>
>